### PR TITLE
Fix error in setup.py  No module named 'pip.req'

### PR DESCRIPTION
**Pull request by apg360**
Fix the error as seen here : https://stackoverflow.com/questions/25192794/no-module-named-pip-req

## Description
https://stackoverflow.com/questions/25192794/no-module-named-pip-req
Before = pip.req import parse_requirements
After =
try: # for pip >= 10
    from pip._internal.req import parse_requirements
except ImportError: # for pip <= 9.0.3
    from pip.req import parse_requirements

## Related Issue
Link to the issue here: ["Error when initial setup #933"](https://github.com/scorelab/Bassa/issues/933)

## Motivation and Context
For a successful setup, otherwise unable to setup Bassa

## How Has This Been Tested?
After fix, "python3 setup.py develop" is able to finish successfully

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

**Review by JaDogg — APPROVED** [review]

**Comment by JaDogg** [comment]
Looks reasonable. Let's wait for another review.

**Comment by kmehant** [comment]
@apg360 can you point your pull request to the `develop` branch as the `master` branch is not updated

**Comment by apg360** [comment]
> @apg360 can you point your pull request to the `develop` branch as the `master` branch is not updated

Done, thank you.

**Pull request closed without merging**
